src/common/logger/Logger.ts

```diff
@@ -1,26 +1,34 @@
-const consoleLogFunctionNames = [
-    'log',
-    'info',
-    'warn',
-    'error',
-    'debug',
-];
+import { Service } from 'typedi';
+import * as winston from 'winston';
+import { config } from './config/default';
```

Comment:
**BUG** gdy jest DI to zaleznosc od konfiguracji powinna byc wstrzykiwana w konstruktorze (ConfigProvider)